.gitmodules

```diff
@@ -2,19 +2,15 @@
 	path = i18n
 	url = https://github.com/mozilla-l10n/mozilla-vpn-client-l10n.git
 	branch = main
-	ignore=all
```

Comment:
I'd like to keep this. Given we update this __everytime__ we build, this should not be commited.

Reply:
That is not what this line does.

If you set `ignore=all` then when you type `git status` you do not see that there have been changes to the submodule.

If you then type `git add .` you will still add the i18n module.

The intention of removing `ignore=all` is to try and help folks not accidentally commit i18n updates by making it clearer that that change has happened locally.